zap/zap.gradle.kts

```diff
@@ -206,23 +219,28 @@ val japicmp by tasks.registering(JapicmpTask::class) {
         "org.parosproxy.paros.core.scanner.VariantAbstractQuery#setParams(int,java.util.Map)",
         "org.parosproxy.paros.db.RecordAlert#getReliability()",
         "org.parosproxy.paros.db.RecordAlert#setReliability(int)",
-        "org.parosproxy.paros.db.paros.ParosTableHistory#getHistoryList(long,int)",
         "org.parosproxy.paros.db.paros.ParosTableHistory#getHistoryList(long)",
+        "org.parosproxy.paros.db.paros.ParosTableHistory#getHistoryList(long,int)",
         "org.parosproxy.paros.db.paros.ParosTableHistory#setHistoryTypeAsTemporary(int)",
         "org.parosproxy.paros.db.paros.ParosTableHistory#unsetHistoryTypeAsTemporary(int)",
         "org.parosproxy.paros.extension.ExtensionPopupMenuItem#isSuperMenu()",
+        "org.parosproxy.paros.extension.history.ExtensionHistory#clearLogPanelDisplayQueue()",
         "org.parosproxy.paros.extension.history.LogPanel#LogPanel()",
+        "org.parosproxy.paros.extension.history.LogPanel#clearDisplayQueue()",
         "org.parosproxy.paros.extension.history.LogPanel#setDisplayPanel(org.zaproxy.zap.extension.httppanel.HttpPanel,org.zaproxy.zap.extension.httppanel.HttpPanel)",
         "org.parosproxy.paros.extension.option.OptionsParamView#getShowMainToolbar()",
         "org.parosproxy.paros.extension.option.OptionsParamView#setShowMainToolbar(int)",
+        "org.parosproxy.paros.model.Session#addGlobalExcludeURLRegexs(java.lang.String)",
+        "org.parosproxy.paros.model.Session#setGlobalExcludeURLRegexs(java.util.List)",
         "org.parosproxy.paros.network.ConnectionParam#getProxyChainSkipName()",
         "org.parosproxy.paros.network.ConnectionParam#setProxyChainSkipName(java.lang.String)",
         "org.parosproxy.paros.view.AbstractFrame#loadIconImages()",
-        "org.parosproxy.paros.view.MainFrame#changeDisplayOption(int)",
         "org.parosproxy.paros.view.MainFrame#MainFrame(int)",
+        "org.parosproxy.paros.view.MainFrame#changeDisplayOption(int)",
```

Comment:
Why this change? There are other changes that make them not to be in alphabetic order.

Reply:
ohh I sorted them with the IDE, I will make changes manually now.

Reply:
the sort was case sensitive, everything is cool now.